.ci/aarch64/clean_up_aarch64.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+stale_process_union=( "docker-containerd-shim" "qemu-system-aarch64" )
+stale_docker_mount_point_union=( "/var/lib/docker/containers" "/var/lib/docker/overlay2" )
+stale_docker_dir="/var/lib/docker"
+stale_kata_dir="/var/lib/vc/sbs"
+
+lib_script="${GOPATH}/src/${tests_repo}/.ci/lib.sh"
+source "${lib_script}"
+
+kill_stale_process()
+{
+	for stale_process in "${stale_process_union[@]}"; do
+		local pid_union=$(ps -eo pid,command | grep "${stale_process}" |  grep -v "grep" | awk '{print $1}')
+		if [ -n "${pid_union}" ]; then
+			while IFS='$\n' read pid; do
+				sudo kill -9 "${pid}"
+			done <<< "${pid_union}"
+		fi
+	done
+}
+
+delete_stale_docker_resource()
+{
+	#before removing stale docker dir, you should umount related resource
+	for stale_docker_mount_point in "${stale_docker_mount_point_union[@]}"; do
+		local mount_point_union=$(mount | grep "${stale_docker_mount_point}" | awk '{print $3}')
+		if [ -n "${mount_point_union}" ]; then
+			while IFS='$\n' read mount_point; do
+				sudo umount "${mount_point}"
+			done <<< "${mount_point_union}"
+		fi
+	done
+	if [ -d "${stale_docker_dir}" ]; then
+                sudo rm -rf "${stale_docker_dir}"
+		systemctl is-active --quiet docker && systemctl restart docker
```

Comment:
I'd probably ensure `docker` is shut down before you start the unmounts and dir removes, and then `start` it again at the end.

Reply:
yep, you're right. I will assure that docker is totally shut down at the first step. maybe `systemctl is-active --quiet docker && systemctl stop docker`

Reply:
Hi~ @grahamwhaley when I firstly want to delete /var/lib/docker, I encountered the` resource busy` err, then it occurred to me that I should kill and unmount relevant resource. I occasionally had the same issue with docker-modification in `manage_ctr_mgr.sh`. which also need to delete relevant dirs.
```
	dir="/var/lib/docker"
	echo "Removing $dir"
```
do you think it also need to check and remove the stale resource like here? 😊